modules/certificate/certificate.resolver.js

```diff
@@ -11,6 +11,9 @@ const certificatesQuery = {
 };
 
 const certificatesMutation = {
+  generateCertificate: async (_, { certificate: { value } }, { user }) =>
+    tryCatchWrapper(certificatesService.generateCertificate(value, user.id)),
```

Comment:
don't use this wrapper now, because its not ready to use

Reply:
Ok